orderer/common/bootstrap/file/bootstrap.go

```diff
@@ -1,119 +0,0 @@
-// Copyright IBM Corp. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-package file
-
-import (
-	"io"
-	"os"
-
-	"github.com/golang/protobuf/proto"
-	cb "github.com/hyperledger/fabric-protos-go/common"
-	"github.com/hyperledger/fabric/orderer/common/bootstrap"
-	"github.com/pkg/errors"
-)
-
-type fileBootstrapper struct {
-	GenesisBlockFile string
-}
-
-// New returns a new static bootstrap helper.
-func New(fileName string) bootstrap.Helper {
-	return &fileBootstrapper{
-		GenesisBlockFile: fileName,
-	}
-}
-
-// NewReplacer returns a new bootstrap replacer.
-func NewReplacer(fileName string) bootstrap.Replacer {
-	return &fileBootstrapper{
-		GenesisBlockFile: fileName,
-	}
-}
-
-// GenesisBlock returns the genesis block to be used for bootstrapping.
-func (b *fileBootstrapper) GenesisBlock() *cb.Block {
-	bootstrapFile, fileErr := os.ReadFile(b.GenesisBlockFile)
-	if fileErr != nil {
-		panic(errors.Errorf("unable to bootstrap orderer. Error reading genesis block file: %v", fileErr))
-	}
-	genesisBlock := &cb.Block{}
-	unmarshallErr := proto.Unmarshal(bootstrapFile, genesisBlock)
-	if unmarshallErr != nil {
-		panic(errors.Errorf("unable to bootstrap orderer. Error unmarshalling genesis block: %v", unmarshallErr))
-	}
-	return genesisBlock
-} // GenesisBlock
-
-// ReplaceGenesisBlockFile creates a backup of the genesis block file, and then replaces
-// it with the content of the given block.
-// This is used during consensus-type migration in order to generate a bootstrap file that
```

Comment:
but what if we decide to have migration from raft to bft? won't we need this then?